mkosi.images/buildernet/mkosi.conf

```diff
@@ -21,6 +21,8 @@ Packages=cryptsetup
          systemd-repart
          systemd-resolved
          tpm2-tools
+         libtss2-esys-3.0.2-0t64
```

Comment:
you can omit the package version. The main mkosi config file already points to a point in time snapshot repository which guarantees stable version of the packages.